app/tut/ga/usaideach.md

```diff
@@ -0,0 +1,339 @@
+
+Seo cúpla sampla conas `athchuir` a úsáid:
+
+{{{
+ár_téacs := "Dia duit :("
+
+>-- Cuir ":)" in ionad ":(" san athróg ár_téacs
+scríobh(athchuir@ár_téacs(":(", ":)"))
+
+>-- Cuir "Slán" in ionad "Dia duit" san athróg ár_téacs
+scríobh(athchuir@ár_téacs("Dia duit", "Slán"))
+}}}
+
+## Roinn
+
+Is féidir linn an gníomh `roinn` a úsáid chun píosa téacs a roinn i [[fo-píosaí|sub pieces]]. Is
+féidir linn carachtar éigin a roghnú agus nuair a glaoimid ar `roinn` leis an gcarachtar, faighimid
+ar ais liosta le gach cuid den píosa téacs atá [[scartha óna gcéile|separated from each other]] leis
+an gcarachtar sin.
+
+Mar shampla: is féidir linn `roinn` a úsáid le [[camóg|comma]] (",") chun píosa téacs cosúil le
+`"Setanta,Python,Java"`{.setanta} a roinn agus an liosta `["Setanta", "Python", "Java"]`{.setanta} a
+fháil. Is féidir linn `roinn` a úsáid le [[spás|space]] (" ") chun abairt éigin a roinn ina
+[[fhocail ar leith|individual words]].
+
+Bain triail as an ríomhchlár seo:
+
+{{{
+>-- Faigh ainm iomlán an úsáideora
+ainm_iomlán := ceist("Cad é d'ainm iomlán?")
+
+>-- Úsáid roinn le spás chun an ainm iomlán a roinn.
+ainmneacha := roinn@ainm_iomlán(" ")
+
+scríobh("Is é", ainmneacha[0], "do chéadainm")
+scríobh("Is é", ainmneacha[1], "do sloinne")
+}}}
+
+![Taispeántas](assets/roinn-ainm.gif)
+
+## Cuid
+
+Leis an gníomh `cuid`, is féidir linn [[cuid|part]] a bhaint amach as píosa téacs. Glacann an gníomh
+le 2 [[innéacs|index]] mar argóintí agus tugann sé an fo-píosa téacs idir na innéacs sin.
+([[Cuimsíonn sé|It includes]] an chéad innéacs, ach ní chuimsíonn sé an dara cheann).
+
+Mar shampla: Tá `cuid@"Setanta"(1, 4)`{.setanta} cothrom le `"eta"`{.setanta} mar is é "eta" an
+téacs idir an innéacs 1 agus 4 i "Setanta". (S**eta**nta).
+
+## go_liosta
+
+Glacann an gníomh `go_liosta` le píosa téacs, agus tugann sé liosta ar ais le gach carachtar ón
+bpíosa téacs ar leith. Mar shampla: Tá `go_liosta("Setanta")`{.setanta} cothrom le `["S", "e", "t",
+"a", "n", "t", "a"]`{.setanta}.
+
+# Liostaí
+
+Oibríonn roinnt gníomhartha téacs le liostaí freisin, mar shampla `cuid`, ach tá roinnt gníomhartha
+[[breise|extra]] ag liostaí:
+
+## Sórtáil
+
+Is féidir an gníomh `sórtáil` a úsáid chun liosta a sórtáil (
+[[mar a shamhlófá|as you could imagine]]). Cuireann `sórtáil` an liosta in
+[[ord méadaitheach|increasing order]]. Mar shampla tá `sórtáil@[1, 3, 2]`{.setanta} cothrom le `[1,
+2, 3]`{.setanta}.
+
+**NB: Tugann `sórtáil` an liosta sórtáilte ar ais, ach athráionn sé an liosta freisin. Ní cruthaíonn
+sé liosta nua: athráíonn sé ord an liosta go díreach.**
+
+## Nasc
+
+Déanann an gníomh `nasc` an [[rud contrártha|opposite thing]] den gníomh `roinn`. Cruthaíonn sé
+píosa téacs ó liosta.  Glacann sé le carachtar éigin agus cruthaíonn sé píosa téacs ó liosta, leis
+an gcarachtar sin idir gach ball den liosta.
+
+Mar shampla: tá `nasc@[1, 2, 3](", ")`{.setanta} cothrom le `"1, 2, 3"`{.setanta}.
```

Comment:
Seachas "cothrom le" is dóigh liom go mbeadh sé níos fearr "Is é "1,2,3," an toradh ar nasc@[1, 2, 3]"

Reply:
Athraithe